packages/commons-server/src/libs/server/crud.ts

```diff
@@ -28,19 +32,20 @@ export const crudRouteParamName = 'id';
  * @param request
  * @returns
  */
-const findItemIndex = (databucketValue: any, request: Request) =>
+const findItemIndex = (
+  databucketValue: any,
+  request: Request,
+  routeCrudKey: RouteResponse['crudKey']
+) =>
   databucketValue.findIndex((value: any, index: number) => {
     if (typeof value === 'object' && value !== null) {
-      if (value[crudRouteParamName] == null) {
+      if (routeCrudKey == null) {
         return false;
       }
 
-      return (
-        value[crudRouteParamName].toString() ===
-        request.params[crudRouteParamName]
-      );
+      return value[routeCrudKey].toString() === request.params[routeCrudKey];
```

Comment:
Use `.id` as we hardcode it to avoid a server restart when changing the `crudKey`

```suggestion
      return value[routeCrudKey].toString() === request.params.id;
```